components/form/ApplicantForm.tsx

```diff
@@ -38,7 +39,19 @@ export default function ApplicantForm({ onSubmit, loading }: { onSubmit: (event:
         </div>
       </div>
 
-      <div className="d-flex justify-content-end">
+      <div className="pb-2 mt-2">
+        <div className="form-floating text-start">
+          <div className="form-check">
+            <input id="consent" name="consent" type="checkbox" className="form-check-input" disabled={loading} required />
+            {/* eslint-disable-next-line jsx-a11y/label-has-associated-control */}
+            <label htmlFor="consent" className="form-check-label">
+              I agree read and agree to the <PrivacyPolicyButtonModal>privacy policy</PrivacyPolicyButtonModal>
```

Comment:
Maybe you meant:

`I have read and agree to the <PrivacyPolicyButtonModal>privacy policy</PrivacyPolicyButtonModal>.`